sdgym/benchmark.py

```diff
@@ -523,3 +534,122 @@ def benchmark_single_table(synthesizers=DEFAULT_SYNTHESIZERS, custom_synthesizer
         write_csv(scores, output_filepath, None, None)
 
     return scores
+
+
+def _create_sdgym_script(params, output_filepath):
```

Comment:
minor: for both of these helpers, can we move them above `benchmark_single_table`?